src/explore-education-statistics-common/src/modules/table-tool/components/Footnote.tsx

```diff
@@ -1,29 +1,20 @@
 import React from 'react';
 import { FullTableMeta } from '@common/modules/full-table/types/fullTable';
-import Details from '@common/components/Details';
+import CollapsibleList from '@common/components/CollapsibleList';
 
 export interface FootnoteProps {
   content: FullTableMeta['footnotes'];
 }
 
 const Footnote = ({ content }: FootnoteProps) => {
-  const footnoteList = content.map(function createFootnotes(footnotes, index) {
-    return (
-      <li key={footnotes.id}>
-        <span className="govuk-!-font-weight-bold">{index + 1}. </span>
-        {footnotes.label}
-      </li>
-    );
+  const footnoteList = content.map(function createFootnotes(footnotes) {
```

Comment:
- Use a lambda instead of named function.
- Inline this as`CollapsibleList`s children